src/main/resources/default/templates/wondergem/scripts/autocomplete.html.pasta

```diff
@@ -90,26 +105,49 @@
                     return true;
                 }
 
-                if (event.keyCode === sirius.keys.KEY_ARROW_UP && completions.selectedRow !== undefined) {
-                    var prevRow = completions.selectedRow.prevAll(".autocomplete-row-js").first();
+                if (event.keyCode === sirius.keys.KEY_ARROW_UP) {
+                    if (completions.selectedRow !== undefined) {
```

Comment:
Do you need to exclude "null" as a value here? if not you could write `if (completions.selectedRow) {` or add a simple helper function like `function exists(object) { return typeof object !== 'undefined' && object !== null; }`